shell/monitor/monitor.sh

```diff
@@ -34,9 +39,71 @@ _monitor_set_experiment_info_func(){
   _monitor_experiment_info_func=( "$@" )
 }
 
+_monitor_login(){
+
+  local output code
+
+  rm -f "$_monitor_cookie_file"
+
+  output=$(curl -sS -w '\n%{http_code}\n' -XPOST https://${_monitor_eshost}/auth/log_in -H 'Content-type: application/json' \
+                -c "$_monitor_cookie_file" -b "$_monitor_cookie_file" -d '{"email": "'"$_monitor_proxy_email"'", "password": "'"$_monitor_proxy_password"'" }')
+
+  code=$( echo "$output" | tail -n1)
+
+  if [ "$code" = "200" ]; then
+    return 0
+  else
+    return 1
+  fi
```

Comment:
This whole conditional block can be replaced with `[ "$code" = "200" ]`, which will return success or failure all by itself. If you think it's more readable to explicitly `return`, that's fair enough.